pkg/api/handlers/generic/containers.go

```diff
@@ -136,7 +135,7 @@ func WaitContainer(w http.ResponseWriter, r *http.Request) {
 	// /{version}/containers/(name)/wait
 	exitCode, err := utils.WaitContainer(w, r)
 	if err != nil {
-		msg = err.Error()
```

Comment:
Shouldn't err be reported somewhere?

Reply:
the error is sent down the pipe in utils.WaitContainer ...